charts/deepgram-self-hosted/samples/04-voice-agent-aws.values.yaml

```diff
@@ -0,0 +1,164 @@
+    # cartesiaSecretRef: "cartesia-api-key"
+    # xaiSecretRef: "xai-secret-ref"
+
+scaling:
+  replicas:
+    api: 1
+    engine: 1
+  auto:
+    # Can toggle to true to enable autoscaling. Make sure to set a value for one of the available metrics
+    enabled: false
+    engine:
+      metrics:
+        speechToText:
+          batch:
+            requestsPerPod: # Discuss a reasonable value with your Deepgram Account Representative
+          streaming:
+            requestsPerPod: # Discuss a reasonable value with your Deepgram Account Representative
+        textToSpeech:
+          batch:
+            requestsPerPod: # Discuss a reasonable value with your Deepgram Account Representative
+        # Discuss a reasoanble value with your Deepgram Account Representative
+        # Must also set engine.concurrencyLimit.activeRequests if using request ratio for autoscaling
+        requestCapacityRatio:
+
+agent:
+  enabled: true
+
+api:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: k8s.deepgram.com/node-type
+                operator: In
+                values:
+                  - api
+  resources:
+    requests:
+      memory: "4Gi"
+      cpu: "2000m"
+    limits:
+      memory: "8Gi"
+      cpu: "4000m"
+
+engine:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: k8s.deepgram.com/node-type
+                operator: In
+                values:
+                  - engine
+  resources:
+    requests:
+      memory: "28Gi"
+      cpu: "6000m"
+      gpu: 1
+    limits:
+      memory: "40Gi"
+      cpu: "8000m"
+      gpu: 1
+  # Discuss a reasonable value with your Deepgram Account Representative
+  # If not using autoscaling, can be left empty, but must be set if using
+  # autoscaling with scaling.auto.engine.metrics.requestCapacityRatio
+  concurrencyLimit:
+    activeRequests:
+
+  modelManager:
+    volumes:
+      aws:
+        efs:
+          enabled: true
+          fileSystemId: fs-xxxxxxxxxxxxxxxx # Replace with your EFS ID
+    models:
+      add:
+        # Recommended Voice Agent models:
+        # - Nova-3 non-formatted (nova-3-general.en.non-formatted.streaming.bf05427e.dg)
+        # - EOT (end-of-turn.4e9dbd96.dg)
+        # - Phonemizer for TTS (phonemizer.en.d6b10ae6.dg)
+        # - Aura-2 TTS voice model(s) (e.g. aura-asteria-en.ecb76e9d.dg)
+        # - Any other streaming STT models of choice (e.g. Nova-2 instead of Nova-3)
+        # - https://link-to-model-1.dg
+        # ...
+      remove:
+        # - https://link-to-old-model-1.dg # Replace these with identifiers for any models already present
+        # - https://link-to-old-model-2.dg #   in the EFS that you'd like removed. For a new installation,
+        # - name-of-old-model-3.dg #   this will likely be empty.
```

Comment:
```suggestion
        # - https://link-to-old-model-1.dg # Replace these with identifiers for any models already present
        # - https://link-to-old-model-2.dg #   in the EFS that you'd like removed. For a new installation,
        # - name-of-old-model-3.dg          #   this will likely be empty.
```